packages/loot-core/src/server/budget/cleanup-template.ts

```diff
@@ -35,12 +37,26 @@ async function processCleanup(month: string): Promise<Notification> {
           sheetName,
           `budget-${category.id}`,
         );
-        await setBudget({
-          category: category.id,
-          month,
-          amount: budgeted - balance,
-        });
-        num_sources += 1;
+        if (balance > 0) {
```

Comment:
Would it make sense to have this be >= so that way exact zeros just act like normal and don't get the warning message?